lightning_pose/data/utils.py

```diff
@@ -358,7 +358,6 @@ def generate_heatmaps(
     width: int,  # width of full sized image
     output_shape: Tuple[int, int],  # dimensions of downsampled heatmap
     sigma: Union[float, int] = 1.25,  # sigma used for generating heatmaps
-    normalize: bool = True,
```

Comment:
did we agree to remove the normalize argument @themattinthehatt @colehurwitz ?

Reply:
we discussed this after you left the meeting last week - I don't see any reason why we should have this option available. it isn't used anywhere else. if we need to resurrect it at some point that's fine but for now it's just clutter.